src/util/key.rs

```diff
@@ -235,7 +235,7 @@ impl FromStr for PublicKey {
 }
 
 /// A Bitcoin ECDSA private key
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash)]
```

Comment:
This is dangerous, see https://github.com/rust-bitcoin/rust-secp256k1/issues/471

Reply:
Will remove, cheers.